Modules/_testcapi/complex.c

```diff
@@ -1,6 +1,9 @@
 #include "parts.h"
 #include "util.h"
 
+#define Py_BUILD_CORE
+#include "pycore_complexobject.h"
```

Comment:
Would it be possible to move `_Py_cr` tests to `Modules/_testinternalcapi/complex.c`? You need to create the new file and add it to `Modules/Setup.stdlib.in` (at `@MODULE__TESTINTERNALCAPI_TRUE@_testinternalcapi` line).

Reply:
Thank, moved.